README.md

```diff
@@ -46,6 +46,14 @@ $ make serve
 
 The application will be served at http://localhost:3000.
 
+To serve the API locally, use:
```

Comment:
Might be good to add that this is when you run the API outside of docker since you can also do Make Serve which uses docker 🙂